flink-table-store-core/src/test/java/org/apache/flink/table/store/table/ChangelogWithKeyFileMetaFilterTest.java

```diff
@@ -46,7 +46,7 @@ public void before() throws Exception {
     @Test
     @Override
     public void testTableScan() throws Exception {
-        writeAndCheckFileMeta(
+        writeAndCheckFileResult(
```

Comment:
Why need to override three methods? I think we can add comments to explain in the code.